content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -98,6 +98,28 @@ Used on: Pod
 This annotation is used to set [Pod Deletion Cost](/docs/concepts/workloads/controllers/replicaset/#pod-deletion-cost)
 which allows users to influence ReplicaSet downscaling order. The annotation parses into an `int32` type.
 
+### kubernetes.io/ingress-bandwidth
+
+Example: `kubernetes.io/ingress-bandwidth: 10M`
+
+Used on: Pod
+
+You can apply quality-of-service traffic shaping to a pod and effectively limit its available bandwidth.
+Ingress traffic (to the pod) is handled by shaping queued packets to effectively handle data.
+To limit the bandwidth on a pod: write an object definition JSON file, and specify the data traffic
+speed using `kubernetes.io/ingress-bandwidth` annotation.
```

Comment:
We should say what the units are. I presume either “bits per second” or “bytes per second” but I wouldn't be 100% sure just from this text.

Reply:
We should mention that these annotations only work if you use a CNI plugin that honors them. I think they might actually be experimental.

Reply:
Here's the reference in kubernetes.io that currently lists this as an experimental feature
https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#support-traffic-shaping

Since its an experimental feature, there should be a note stating so

Reply:
@sftim @reylejano changes done🙂